### ISSUE TITLE
🚀 [Feature] Set the focus for HTML form elements

### ISSUE THREAD

**Issue by sushidave** [Ocelot-Social-Community/Ocelot-Social#4652]
## :rocket: Feature Request

Consequently set the focus for HTML form elements on page load. 

This feature improves UX as it saves time and effort (key strokes/clicks/taps) for users and facilitates auto-typing for password managers. 

Example:

Actual form:
![focus-current](https://user-images.githubusercontent.com/17728384/132088722-48355921-14ab-49bd-aa5e-293d78be5cd4.png)

Target form:
![focus-future](https://user-images.githubusercontent.com/17728384/132088730-e85f4306-c75a-4d21-b06b-2fc14053e5e7.png)

Applies to:
- Registration form: text input field on each page
- Login form: first text input field
- Forgot password form: text input field
- Post feed: search field
- New post: first text input field
- Settings - Your data: first input field
- Settings - Your e-mail: input field
- Settings - Security: first input field
- Settings - Privacy: checkbox
- Settings - Social media: input field
- Settings - Delete user account: input field

Focus should also be applied to forms which are not listed here and to future forms.

**Comment by AayushSaini101** [comment]
Can i work on this issue

**Comment by sushidave** [comment]
@AayushSaini101 Hi, thanks for your interest and welcome to this project. Yes you can. @Tirokk is going to provide you with more information soon.

**Comment by AayushSaini101** [comment]
Thanks please assign me this issue

**Comment by maddler-hub** [comment]
Can I work on this issue??

**Comment by ismekashif** [comment]
Hi! I would like to work on this.

**Comment by sushidave** [comment]
Hi @ismekashif! Thanks for your interest in working on this issue and welcome to this project! @Tirokk will give you further info soon.

**Comment by Tirokk** [comment]
Hey @ismekashif ,

yes for sure you can work on this. That’s great! 🤗
I assigned you to this issue.

I cared for your invitation to our community as volunteer via an e-mail invitation you can accept.

Our conversation runs over our Discord server:
https://discord.gg/AJSX9DCSUA
There we have usually from Monday to Thursday a Daily Standup at CET 11:30 am (UTC 10:30 am) in the Office Cube.

Please don’t compare from your fork to this repository, in case, because the publish action will fail, because of permission problems with the credentials.
Just push your branch - directly to our repo and compare to the master.
If you accept the invitation then you’ll have the right todo so. 😉

**Comment by Tirokk** [comment]
If you need any help @ismekashif just let me know and we can have a quick pairing session … 🤗

**Comment by ismekashif** [comment]
Sure, Thanks! @Tirokk

**Comment by ismekashif** [comment]
Hi! @Tirokk, I think I might need your help with docker. I couldn't get it to run. (This is my first time with docker)

**Comment by Tirokk** [comment]
@ismekashif I wrote you on Discord. 🤗

**Comment by Tirokk** [comment]
Any problems by working on this issue? @ismekashif 

PS: No pressure, just try to help …

**Comment by ismekashif** [comment]
> Any problems by working on this issue? @ismekashif 
> 
> PS: No pressure, just try to help …

All working fine except that I have to restart that particular container everytime I have made changes to see if it works. And that is taking a lot of time.

**Comment by Tirokk** [comment]
Usually the container has not to be restarted. I just tried it out. 🤔
To always restart the containers is unbearable.

May we should install the things locally? @ismekashif

**Comment by ismekashif** [comment]
> Usually the container has not to be restarted. I just tried it out. 🤔
> To always restart the containers is unbearable.
> 
> May we should install the things locally? @ismekashif 

Yes, it is getting quite unbearable. How does installing things locally work?

**Comment by Tirokk** [comment]
Have a look here: https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/SUMMARY.md

And especial in the [backend](https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/backend/README.md)
And in the [webapp](https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/webapp/README.md)

To morrow I didn’t work.
Will be back on Montag in case you need some help.

**Comment by Tirokk** [comment]
@ismekashif I learned from @Mogge last week that it is somehow easy just to start the Neo4j database in Docker and the remaining apps locally. That works well he said.

He says that there are general problem with running Docker on Windows, which seem not to be solved satisfying by Docker.

**Comment by ismekashif** [comment]
@Tirokk Thanks for letting me know. It's such a relief to know that it is not my machine. I will try it and let you know.

**Comment by Tirokk** [comment]
Hey @ismekashif ,

there’s a solution in the middle:

See issue #4793

**Comment by ismekashif** [comment]
> Hey @ismekashif ,
> 
> there’s a solution in the middle:
> 
> See issue #4793

I have seen that issue, but thanks for pointing it out

> I can help you with it, if you like …

Will try it first by myself and see, if I can't get it to work then I will ask you for your help

**Comment by ismekashif** [comment]
I tried following that method but `yarn install` gave error in webapp and did not work. I was working on another project and had the same problem of HMR. I searched the internet for the problem and got the solution. The repo should be in the user's folder of the windows subsystem for linux. Now HMR works for me using docker. @Tirokk

**Comment by Tirokk** [comment]
Thanks a lot for letting us know @ismekashif 🙏🏼😍

> I tried following that method but `yarn install` gave error in webapp and did not work. I was working on another project and had the same problem of HMR. I searched the internet for the problem and got the solution. The repo should be in the user's folder of the windows subsystem for linux. Now HMR works for me using docker. @Tirokk

I noted your solution here: https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/4793#issue-1059990765
